modules/custom/d_update/src/Updater.php

```diff
@@ -428,23 +429,38 @@ public function updateConfigurations($source, $name) {
     }
     foreach ($data as $configName => $configOperations) {
       $updates = $configOperations;
-      $newConfig = [];
+      $config = $this->configFactory->getEditable($configName);
+      $newConfig = $config->get();
       $isOptional = $updates['optional'] ?? FALSE;
 
       if (isset($updates['delete'])) {
-        foreach ($updates['delete'] as $value) {
-          NestedArray::unsetValue($newConfig, explode('.', $value));
+        foreach ($updates['delete'] as $update) {
+          NestedArray::unsetValue($newConfig, explode(':', $update));
```

Comment:
a tak w sumie, to dlaczego zmieniasz `.` na `:` ? dla czytelnosci?

Reply:
w cfg od update podajesz nazwy parentów, dla przykładu:
chcesz usunąć
```
content:
  field_media_image:
    settings:
      responsive_image_style: {  }
```
bardziej czytelne i zrozumiałe wydaje mi się wpisanie
```
core.entity_view_display.media.d_image.d_product_image:
  delete:
    - content:field_media_image:settings:responsive_image_style
```
niż
```
core.entity_view_display.media.d_image.d_product_image:
  delete:
    - content.field_media_image.settings.responsive_image_style
```